### ISSUE TITLE
Latest version can crash Chromium on fully up-to-date Arch Linux

### ISSUE THREAD

**Issue by daniel3735928559** [daniel3735928559/guppy#36]
**Steps performed:** 

* On chromium 54.0.2840.100 (64-bit) on Arch Linux (fully upgraded system), go to the [demo page](https://daniel3735928559.github.io/guppy/)
* Enter `x ^ 2`
* Hit `backspace` twice

**Expected result:**

Left with `x^[?]` with the cursor to the right of the `x`.

**Observed result:**

The page crashes with the error "Aw, Snap!  Something went wrong while displaying this webpage".

**Comment by daniel3735928559** [comment]
This appears to be an XSLTProcessor issue in chromium, as it fails on the line 

```
    var tex_doc = Guppy.xsltProcessor.transformToDocument(base);
```

**Comment by daniel3735928559** [comment]
Issue not present in Firefox nor with the offending XML document tested with `xsltproc` in the command line

**Comment by daniel3735928559** [comment]
This is a vexing issue, as the particular combination of XML document and XSL transformation that triggers it is unclear.  So we temporarily give up on finding a workaround, and have instead implemented in 73ab480f2bc0f5d83da5b74d0f7f7b2fec7791fd a way to render without XSLT entirely.  This new method seems to produce identical results to the XSLT rendering method and passes all tests, and shouldn't be slower, so we will cautiously proceed with this for the moment.
